package.json

```diff
@@ -9,22 +9,23 @@
     "node": ">=16"
   },
   "peerDependencies": {
-    "@checkdigit/eslint-plugin": "^4.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.33.0",
-    "@typescript-eslint/parser": "^5.33.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-deprecate": "^0.7.0",
-    "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-no-only-tests": "^3.0.0",
-    "eslint-plugin-no-secrets": "^0.8.9",
-    "eslint-plugin-sonarjs": "^0.15.0",
-    "eslint-plugin-unicorn": "^43.0.2"
+    "@checkdigit/eslint-plugin": "4.0.0",
+    "@typescript-eslint/eslint-plugin": "5.33.1",
+    "@typescript-eslint/parser": "5.33.1",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-deprecate": "0.7.0",
+    "eslint-plugin-eslint-comments": "3.2.0",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-no-only-tests": "3.0.0",
+    "eslint-plugin-no-secrets": "0.8.9",
+    "eslint-plugin-sonarjs": "0.15.0",
+    "eslint-plugin-unicorn": "43.0.2"
   },
   "scripts": {
+    "preversion": "npm test",
```

Comment:
any particular reason to bring this back?

Reply:
um.. no?  :). removed